src/writeData/subscriptions/components/LineProtocolForm.tsx

```diff
@@ -1,38 +1,79 @@
 // Libraries
-import React, {FC} from 'react'
+import React, {FC, useState} from 'react'
 
 // Components
-import {Grid, Heading, HeadingElement, FontWeight} from '@influxdata/clockface'
+import {Grid, Form, Dropdown, ComponentStatus} from '@influxdata/clockface'
+
+// Types
+import {Subscription} from 'src/types/subscriptions'
+
+// Utils
+import {event} from 'src/cloud/utils/reporting'
 
 // Styles
 import 'src/writeData/subscriptions/components/LineProtocolForm.scss'
 
-const LineProtocolForm: FC = () => {
-  // const [lp, setLineProtocol] = useState('')
+interface Props {
+  formContent: Subscription
+  edit: boolean
+}
+
+const LineProtocolForm: FC<Props> = ({formContent, edit}) => {
+  const microsecondsType = 'MS'
+  const secondsType = 'S'
+  const microseconds2Type = 'US'
+  const nanosecondsType = 'NS'
+  const precisionList = [
+    nanosecondsType,
+    microsecondsType,
+    secondsType,
+    microseconds2Type,
+  ]
```

Comment:
I see we're defining this repeatedly - could it be abstracted somewhere? Maybe stored in a type within a context?

Reply:
I'll pull it out!